lms/migrations/versions/9ee0679761f0_.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
Migrations are not great here, they need to be broken into different steps to deploy